python/paddle/fluid/distribute_transpiler.py

```diff
@@ -500,38 +484,83 @@ def _clone_var(self, block, var):
             lod_level=var.lod_level,
             persistable=True)
 
-    def _append_split_op(self, program, gradblocks):
+    def _find_op_by_out(self, program, var):
+        for idx, op in enumerate(program.global_block().ops):
+            if var.name in op.output_arg_names:
+                return idx + 1
+        return -1
+
+    def _insert_send_vars_op(self, program, index, send_vars, epmap, eps):
+        rpc_client_var = layers.io.get_rpc_client_var(program)
+
+        program.global_block().insert_op(
+            index=index,
+            type="send_vars",
+            inputs={"X": send_vars},
+            outputs={"RPCClient": rpc_client_var},
+            attrs={"sync_send": 0,
+                   "epmap": epmap,
+                   "endpoints": eps})
+
+    def _append_trainer_op(self, program, gradblocks, spliter):
```

Comment:
I think "trainer op" is too general, the trainer could do forward, calculate grad, send grad, recv param. But this function seems to only do split and send grad? Maybe change to another more specific name?